meta-resin-common/recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -10,6 +10,10 @@ SRC_URI_append = " \
     file://resin-sample \
     "
 
+# do a build in the source directories (by doing so we avoid a bunch of errors when building outside the source directory)
+# (the git version of NM already can handle build_dir != source_dir so most likely we need to check and maybe remove this on the next stable version of NM)
```

Comment:
Issues that could benefit from getting back to in some time should be marked with XXX as described here: https://en.wikipedia.org/wiki/Comment_(computer_programming)#Tags